src/components/ProductsEmptyState/tests/ProductsEmptyState.test.tsx

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+import { vi } from 'vitest';
+
+import { categoriesMock } from '@/mocks/categories';
+import { renderWithRedux } from '@/testsUtils/providers';
+import { screen, cleanup } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import ProductsEmptyState from '..';
+
+vi.mock(`@clerk/nextjs`, () => ({
+  __esModule: true,
+  useUser: () => ({
+    user: {
+      id: `1`,
+    },
+  }),
+}));
+
+vi.mock(`axios`);
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe(`ProductsEmptyState`, () => {
+  afterAll(() => {
+    vi.clearAllMocks();
+  });
+
+  afterEach(() => {
+    cleanup();
+    vi.resetAllMocks();
+  });
+
+  it(`should open create category modal when clicking on button "Cadastrar categoria"`, async () => {
```

Comment:
```suggestion
  it(`should open create product modal when clicking on button "Cadastrar produto"`, async () => {
```
I think that the description should be like this, no?